interface.yaml

```diff
@@ -2,3 +2,5 @@ name: kubernetes-cni
 summary: Interface for relating various CNI implementations
 version: 0
 maintainer: "Rye Terrell <rye.terrell@canonical.com>"
```

Comment:
@Cynerva since you're up in here, mind taking over maintainership?

Reply:
It's mine now.